src/Components/access-requests-wizard/AccessRequestsWizard.js

```diff
@@ -0,0 +1,284 @@
+        }}
+        actions={[
+          <Button
+            key="confirm"
+            variant="primary"
+            onClick={() => onClose(false)}
+          >
+            Exit
+          </Button>,
+          <Button
+            key="cancel"
+            variant="link"
+            onClick={() => setCancelWarningVisible(false)}
+          >
+            Stay
+          </Button>,
+        ]}
+      >
+        All inputs will be discarded.
+      </Modal>
+    );
+  } else if (error || isLoading || isSubmitting) {
+    content = (
+      <Modal
+        isOpen
+        variant={ModalVariant.large}
+        showClose={false}
+        className="accessRequests"
+        hasNoBodyWrapper
+        aria-describedby="wiz-modal-description"
+        aria-labelledby="wiz-modal-title"
+      >
+        <Wizard
+          className="accessRequests"
+          title={`${capitalize(variant)} request`}
+          steps={[
+            {
+              name: 'status',
+              component: isLoading ? (
+                <Bullseye>
+                  <Spinner />
+                </Bullseye>
+              ) : (
+                <EmptyState>
+                  {isSubmitting ? (
+                    <>
+                      <EmptyStateIcon icon={() => <Spinner size="lg" />} />
+                      <Title headingLevel="h2" size="lg">
+                        Submitting access request
+                      </Title>
+                      <Button variant="link" onClick={() => onClose(true)}>
+                        Close
+                      </Button>
+                    </>
+                  ) : (
+                    <>
+                      <EmptyStateIcon
+                        icon={ExclamationCircleIcon}
+                        color="#C9190B"
+                      />
+                      <Title headingLevel="h2" size="lg">
+                        {error.title || 'An error occured'}
+                      </Title>
+                      <EmptyStateBody>{error.description}</EmptyStateBody>
+                      {error.title !== invalidAccountTitle && (
+                        <Button variant="primary" onClick={() => setError()}>
+                          Return to Step 1
+                        </Button>
+                      )}
+                    </>
+                  )}
+                </EmptyState>
+              ),
+              isFinishedStep: true,
+            },
+          ]}
+          onClose={() => onClose(true)}
+        />
+      </Modal>
+    );
+  } else {
+    content = (
+      <FormRenderer
+        schema={schema.current}
+        subscription={{ values: true }}
+        FormTemplate={FormTemplate}
+        componentMapper={{ ...componentMapper, ...mapperExtension }}
+        validatorMapper={validatorMapper}
+        onSubmit={onSubmit}
+        initialValues={{
+          'first-name': user?.first_name,
+          'last-name': user?.last_name,
+          ...initialValues.current,
```

Comment:
Do we want to potentially override `first-name` and `last-name` with values from the `initialValues`? Or these fields are never in the values object?

Reply:
These values should not be there. However, changed the order of the lines to potentially avoid it.